dist/package.json

```diff
@@ -0,0 +1,93 @@
+{
+  "$schema": "./node_modules/ng-packagr/package.schema.json",
+  "name": "@ollca/ng-lazyload-image",
```

Comment:
This file should not be committed.

Reply:
It should even be possible (without force).
https://github.com/tjoskar/ng-lazyload-image/blob/2374cbf6d827d95e46b9a9cb9bc66632f08bcdcb/.gitignore#L2

Reply:
Thk, I removed it

Reply:
Regarding your proposition, I am not sure the app will be loaded by the crawler during indexation ? If yes the image should be rendered when `isBoot` function returns `true`. Otherwise, the image url should be present in the HTML return by the server